### PR TITLE
Update evidence create to return verified status.

### PR DESCRIPTION
**Pull request by osaidwtd**

### CODE REVIEW

**Comment by osaidwtd** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by lesnerd** [comment]
recheck

**Comment by osaidwtd** [comment]
recheck

File: evidence/create.go [hunk 1]
```diff
@@ -138,11 +139,21 @@ func (ec *EvidenceCreateCommand) Run() error {
...
+	createResponse := &model.CreateResponse{}
+	err = json.Unmarshal(body, createResponse)
+	if err != nil {
+		return err
+	}
+	if createResponse.Verified {
+		clientlog.Info("Evidence successfully created and verified")
+		return nil
+	}
+	clientlog.Info("Evidence successfully created but not verified")
```
Comment (denys-jfrog) [inline evidence/create.go:156]: Please, check the wording with Michael.

**Review by denys-jfrog — APPROVED** [review]

**Comment by osaidwtd** [comment]
recheck

File: evidence/create.go (hunk 1 above)
Comment (osaidwtd) [reply evidence/create.go:156]: I did

**Comment by osaidwtd** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by lesnerd — APPROVED** [review]

**Comment by osaidwtd** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**